src/lib/Scenes/Fair/Screens/FairBMWArtActivation.tsx

```diff
@@ -0,0 +1,147 @@
+import { Box, Flex, Sans, Serif, space, Theme } from "@artsy/palette"
+import { FairBMWArtActivationQuery } from "__generated__/FairBMWArtActivationQuery.graphql"
+import { CaretButton } from "lib/Components/Buttons/CaretButton"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { FlatList, Image, ViewProperties } from "react-native"
+import { graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components/native"
+import { defaultEnvironment } from "../../../relay/createEnvironment"
+import renderWithLoadProgress from "../../../utils/renderWithLoadProgress"
+
+interface Props extends ViewProperties {
+  fair: FairBMWArtActivationQuery["response"]["fair"]
```

Comment:
The `["response"]["fair"]` is a little confusing to me, but I think it's caused by the Relay issue I mentioned lower in this file. 

Also, I didn't know that TypeScript would let you refer to nested types with this subscript notation – very cool!